### ISSUE TITLE
[Styling] Fix road color in Streets

### ISSUE THREAD

**Issue by lazaa32** [maptiler/qgis-maptiler-plugin#75]
Roads disappear in Streets close to zoom level 12.5

**Comment by lazaa32** [comment]
There are two related styles:
`road_trunk_primary_link_casing`:
```
{5: 'CASE WHEN @zoom_level > 5 AND @zoom_level <= 7 THEN scale_exp(@zoom_level, 5, 7, 0, 0.7, 1.2) * 0.127 WHEN @zoom_level > 7 AND @zoom_level <= 20 THEN scale_exp(@zoom_level, 7, 20, 0.7, 14, 1.2) * 0.127 END'}
```
and 
`road_trunk_primary_casing`:
```
{5: 'CASE WHEN @zoom_level > 5 AND @zoom_level <= 6 THEN scale_exp(@zoom_level, 5, 6, 0.4, 0.7, 1.2) * 0.127 WHEN @zoom_level > 6 AND @zoom_level <= 7 THEN scale_exp(@zoom_level, 6, 7, 0.7, 1.75, 1.2) * 0.127 WHEN @zoom_level > 7 AND @zoom_level <= 20 THEN scale_exp(@zoom_level, 7, 20, 1.75, 22, 1.2) * 0.127 END'}
```
Because of styling `road_trunk_primary_link_casing` appears at zoom 10 but `road_trunk_primary_casing` not until zoom 13.

**Comment by lazaa32** [comment]
Closed in #85.

**Comment by lazaa32** [comment]
Fix in #101.
